### PR TITLE
Fix Bug in call to Symbol Uploader

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Symbol uploader does not tolerate the quotes.

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]

File: eng/pipelines/templates/jobs/mgmt-release.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ stages:
                     inputs:
                       solution: '${{ parameters.BuildToolsPath }}/tools/symboltool/SymbolUploader.proj'
                       msbuildArguments: >
-                        /p:PackagesPath='$(PIPELINE.WORKSPACE)\packages'
+                        /p:PackagesPath=$(PIPELINE.WORKSPACE)\packages
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/mgmt-release.yml:63]: Do you plan to revert the slashes as well now? Also is it just the single quotes issue, should these be double quotes?

Comment (chidozieononiwu) [reply eng/pipelines/templates/jobs/mgmt-release.yml:63]: I can revert the slashes. I think double quotes will be fine but am more confident with passing no quotes as that how we have done it else where.

Comment (weshaggard) [reply eng/pipelines/templates/jobs/mgmt-release.yml]: Passing no quotes works fine as long as the paths have not spaces in them.

**Pull request merged**